packages/plugins/rubygem-foreman-tasks/foreman-tasks.logrotate

```diff
@@ -0,0 +1,9 @@
+/var/log/foreman/*.output {
```

Comment:
do we expect any other files to be called `*.output` besides `dynflow_executor.output`? If we don't, I'd prefer being explicit here and list `/var/log/foreman/dynflow_executor.output`.